ui/src/index.tsx

```diff
@@ -121,6 +122,9 @@ class Root extends PureComponent {
                         <Route path="new" component={CreateOrgOverlay} />
                         <Route path=":orgID" component={SetOrg}>
                           <IndexRoute component={MePage} />
+                          <Route path="" component={MePage}>
+                            <Route path="profile" component={ProfileOverlay} />
```

Comment:
Threw this in here so the profile overlay has something in the background. It's a bit of a workaround because you can't give `IndexRoute` a child

Reply:
Could the `IndexRoute` be removed altogether?
```
 <Route path=":orgID" component={SetOrg}>
   <Route path="" component={MePage}>
     <Route path="profile" component={ProfileOverlay} />
```
Not sure, I'm not very familiar with `IndexRoute`s.

Reply:
I have no idea

Reply:
@bthesorceror thoughts?

Reply:
your overlay route should be a child of the IndexRoute